draft-tigress-requirements.md

```diff
@@ -65,108 +70,98 @@ This document describes the use cases necessitating the secure transfer of digit
 
 # Introduction
 
-In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key to a hotel room or a digital key to a private house) from a wallet on one device (smartphone) to another, particularly, if these devices belong to two different platforms (e.g. one is iOS, another - Android).
-Today, there is no widely accepted way of transferring digital credentials securely between two digital wallets independent of hardware and software manufacturer. This document describes the problem space and the requirements for the solution the working group creates.
+In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key to a hotel room) from one device (like smartphone) to another. Today, there is no widely accepted way of transferring digital credentials securely between two devices from different manufacturers. 
```

Comment:
I personally like the old wording more. I'll leave it to others folks to comment on this change

Reply:
added more reviewers

Reply:
The main idea behind this PR is to make the document more generic.
-- The term "from a wallet" makes it seem very specific to one app on the device.
-- smartphone makes it specific to a certain class of devices.

Also, as we heard in the meeting, cross-platform is a default of any IETF spec. So the specific mention of iOS and Android felt redundant.